restapi/src/main/java/io/stargate/web/docsapi/service/CollectionService.java

```diff
@@ -52,6 +54,6 @@ public boolean upgradeCollection(
     if (upgradeType == CollectionUpgradeType.SAI_INDEX_UPGRADE) {
       return docDB.upgradeTableIndexes(keyspaceName, tableName);
     }
-    throw new RuntimeException("Invalid upgrade type: " + upgradeType);
+    throw new ErrorCodeRuntimeException(ErrorCode.DOCS_API_GENERAL_UPGRADE_INVALID);
```

Comment:
Why not include `upgradeType`? It appears to be set by the user, so it'd be nice to indicate what exactly is not valid.